Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -105,11 +106,28 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
   private var value: Value? = nil
 
   public init(wrappedValue value: Value?) {
+    if let value = value {
+      // TODO: Define better way to use FIRLogger from Swift.
+      FIRLogDebugSwift(
+        "[FirebaseFirestoreSwift]",
+        "I-FST000001",
```

Comment:
This code already exists: Firestore/core/src/util/log_apple.mm:  FIRLogBasic(ToFIRLoggerLevel(level), kFIRLoggerFirestore, @"I-FST000001",

It looks like Firestore only uses one code, but let's make this one I-FST000002 for now.

Reply:
Done!